functions/__sf_section_kubecontext.fish

```diff
@@ -0,0 +1,32 @@
+#
+#  Kubernetes (kubectl)
+#
+# Kubernetes is an open-source system for deployment, scaling,
+# and management of containerized applications.
+# Link: https://kubernetes.io/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+function __sf_section_kubecontext -d "Display the kubernetes context"
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_SHOW true
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_PREFIX " "
+  __sf_util_set_default SPACEFISH_KUBECONTEXT_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_SYMBOL "☸️"
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_COLOR cyan
+
+	[ $SPACEFISH_KUBECONTEXT_SHOW = false ]; and return
+
+	if not type -q kubectl
+		return
+	end
+
+	set -l sf_kube_context (kubectl config current-context 2>/dev/null)
```

Comment:
To redirect to stderr in Fish, you would replace `2>` with `^` 😄

Reply:
Since this variable is local, I wouldn't use the `sf_` prefix. That has only been used in the past for global variables so that we don't occupy the namespace.